supabase_auth/helpers.py

```diff
@@ -128,8 +136,30 @@ def handle_exception(exception: Exception) -> AuthError:
             return AuthRetryableError(
                 get_error_message(error), error.response.status_code
             )
-        json = error.response.json()
-        return AuthApiError(get_error_message(json), error.response.status_code or 500)
+        data = error.response.json()
+
+        error_code = None
+        response_api_version = parse_response_api_version(error.response)
```

Comment:
Probably this `parse_response_api_version` should return Unix timestamp 0 and your code below will magically work.